swagger/v1/person_escort_record.yaml

```diff
@@ -100,9 +100,8 @@ PersonEscortRecord:
     relationships:
       type: object
       required:
-        - profile
+        - move
         - framework
-        - responses
```

Comment:
How comes we're removing responses?

Reply:
see https://github.com/ministryofjustice/hmpps-book-secure-move-api/pull/1045#discussion_r508295933